migrations.sql

```diff
@@ -77,6 +77,13 @@ CREATE OR REPLACE VIEW routelandia.highwaysHavingStations AS
   GROUP BY highways.highwayid
   HAVING count(distinct stations.stationid)>0;
 
+-------------------
```

Comment:
You did something here I'm pretty sure you didn't mean to do... You both moved and uncommented this function. :-)